canvas_exercise/shapes_game/index.js

```diff
@@ -1,36 +1,123 @@
-window.addEventListener("load", function() {
+//get random shape on screen
```

Comment:
Looks good.  There are a few bugs you can work on.  Whenever I press the space bar to start the game, my score starts out as -1.  It seems to be related to how you attribute points in the keyup callback.  Also, After a game is over, I can't see my score.  Try showing it in the game screen after 30 seconds is up.